### PR TITLE
Add YOLOv8 Oriented Bounding Boxes notebook

### PR DESCRIPTION
**Pull request by capjamesg**
# Description

This PR adds a notebook to accompany our YOLOv8 Oriented Bounding Boxes tutorial.

## Type of change

- [X] New notebook

## How has this change been tested, please provide a testcase or example of how you tested the change?

This notebook has been tested by running all cells on the pre-set solar panel dataset.

## Any specific deployment considerations

N/A

## Docs

N/A

### CODE REVIEW

File: README.md
Comment (stellasphere) [inline README.md:1]: is a link to yolov8 keypoint correct?

**Review by stellasphere — COMMENTED** [review]
Can you confirm that the links are correct? They have the text for OBB, but have links to keypoint notebooks, not the OBB notebooks you made in this branch

File: README.md
Comment (SkalskiP) [reply README.md:1]: Yup, the link in the table is incorrect.

**Comment by SkalskiP** [comment]
The link to the blog post is incorrect. 

Screenshot 2024-02-06 at 22 48 56

**Comment by SkalskiP** [comment]
I would suggest to uncomment `!pip install supervision -q` and pin specific version.

File: README.md
Comment (capjamesg) [reply README.md:1]: This has now been fixed.

**Comment by capjamesg** [comment]
I have made all the requested changes listed above.

**Review by SkalskiP — APPROVED** [review]

**Comment by SkalskiP** [comment]
@capjamesg good work! 🔥

**Pull request merged**